### PR TITLE
add BAYCKRC targets

### PR DESCRIPTION
**Pull request by BAYCKRC**
add BAYCKRC targets.
There is contact with Alka on Discord.

### CODE REVIEW

File: f421makefile.mk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 TARGETS_F421 :=  WRAITH32_F421 AIKON_SINGLE_F421 AIKON_55A_F421 FOXEER_F421 HAKRC_K_F421 \
 	             HAKRC_G_F421 TEKKO32_F421 AT32DEV_F421 DARWIN_F421 NEUTRON_F421 AT32SLOTCAR_F421\
-				 HAKRC_G23_F421 NEUTRON_MINIAIO_F421 FOXEER_F421\  
+				 HAKRC_G23_F421 NEUTRON_MINIAIO_F421 BAYCKRC_S_55A_F421 FOXEER_F421\
```
Comment (tridge) [inline f421makefile.mk:4]: this isn't needed with the new makefile system, can you please rebase the PR on main?

**Review by tridge — CHANGES_REQUESTED** [review]

File: f421makefile.mk (hunk 1 above)
Comment (BAYCKRC) [reply f421makefile.mk:4]: Okay, everything will be adjusted according to your latest coding structure. If you need me to do anything, just give me instructions. Thank you.

Comment (BAYCKRC) [reply f421makefile.mk]: > this isn't needed with the new makefile system, can you please rebase the PR on main?

The modifications in the makefile have been removed.

**Comment by tridge** [comment]
@BAYCKRC you should squash the two commits to one, like this:
https://github.com/tridge/AM32/tree/add_BAYCKRC_targets
that will stop the conflict
unfortunately I need @AlkaMotors or @freasy to run the CI jobs, I don't have workflow approval yet

**Comment by BAYCKRC** [comment]
> @BAYCKRC you should squash the two commits to one, like this: https://github.com/tridge/AM32/tree/add_BAYCKRC_targets that will stop the conflict unfortunately I need @AlkaMotors or @freasy to run the CI jobs, I don't have workflow approval yet

Okay, that's how it needs to be. In the example, your GitHub repository has already created a PR. Can you merge it directly into the main repository?

**Pull request closed without merging**